addons/web/static/src/views/fields/boolean_toggle/boolean_toggle_field.js

```diff
@@ -9,12 +9,21 @@ export class BooleanToggleField extends BooleanField {
         return this.props.record.isReadonly(this.props.name);
     }
     onChange(newValue) {
-        this.props.update(newValue, { save: true });
+        this.props.update(newValue, { save: this.props.autosave });
```

Comment:
Note that if someone adds autosave: false in a kanban view, the widget will be editable (as always), but won't save directly, and there'll be no Save button available for the user. But maybe it's the responsibility of the one that writes the arch.